app/web/features/communities/newEvents/DiscoverEventsList.tsx

```diff
@@ -0,0 +1,212 @@
+  },
+  pagination: {
+    display: "flex",
+    justifyContent: "center",
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+  row: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    width: "100%",
+    [theme.breakpoints.down("sm")]: {
+      flexDirection: "column",
+    },
+  },
+  selectedFilter: {
+    backgroundColor: theme.palette.secondary.main,
+    color: theme.palette.common.white,
+    padding: theme.spacing(1, 2),
+    textAlign: "center",
+    fontWeight: "bold",
+    margin: theme.spacing(0.5),
+    borderRadius: theme.shape.borderRadius * 6,
+    "&:hover": {
+      cursor: "pointer",
+    },
+  },
+}));
+
+const DiscoverEventsList = () => {
+  const classes = useStyles();
+  const { control, errors } = useForm({
+    mode: "onChange",
+  });
+  const { t } = useTranslation([GLOBAL, COMMUNITIES]);
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+  const pageSize = 12;
+
+  const [pageNumber, setPageNumber] = useState(1);
+  const [isMyCommunities, setIsMyCommunities] = useState<boolean>(false);
+  const [isOnlineOnly, setIsOnlineOnly] = useState<boolean>(false);
+  const [locationResult, setLocationResult] = useState<GeocodeResult | "">("");
+
+  const { data, error, isLoading } = useEventSearch({
+    pastEvents: false,
+    pageSize,
+    pageNumber,
+    isMyCommunities,
+    isOnlineOnly,
+    searchLocation: locationResult,
+  });
+
+  const hasEvents = data && data.eventsList && data.eventsList.length > 0;
+  const numPages = Math.ceil((data?.totalItems ?? 0) / pageSize) ?? 1;
+
+  const handlePageNumberChange = (
+    event: React.ChangeEvent<unknown>,
+    value: number
+  ) => {
+    setPageNumber(value);
+  };
+
+  const handleFilterIsMyCommunitiesClick = () => {
+    setIsMyCommunities(!isMyCommunities);
+    setPageNumber(1);
+  };
+
+  const handleFilterIsOnlineOnlyClick = () => {
+    if (isOnlineOnly) {
+      setLocationResult("");
+    }
+    setIsOnlineOnly(!isOnlineOnly);
+    setPageNumber(1);
+  };
+
+  const handleOnChangeAutocomplete = (newLocationResult: GeocodeResult) => {
+    if (typeof newLocationResult === "object") {
+      setIsOnlineOnly(false);
+      setLocationResult(newLocationResult);
+    } else {
+      setLocationResult("");
+    }
+    setPageNumber(1);
+  };
+
+  const renderLocationAutoComplete = () => (
+    <LocationAutocomplete
+      className={classes.locationSearch}
+      control={control}
+      name="location"
+      defaultValue={typeof locationResult === "object" ? locationResult : ""}
+      label={t("global:location_autocomplete.search_location_button")}
+      onChange={handleOnChangeAutocomplete}
+      fieldError={errors.location?.message}
+      fullWidth={isMobile}
+    />
+  );
+
+  return (
+    <>
+      <div className={classes.column}>
+        <Typography variant="h2">
+          {t("communities:discover_events_title")}
```

Comment:
H1 instead? feels a bit small

![image](https://github.com/user-attachments/assets/9618cb1b-53fc-4f74-ba0f-5dee333be982)

Reply:
UI wise and SEO for google crawling wise a page should only have one H1 per page.

I can do H2s but had it originally and they felt too big. Maybe I do the final ticket with tests and I assume Jesse and co will test before release and we can do all the UI preferences then.